src/reference/asciidoc/ip.adoc

```diff
@@ -1358,3 +1358,101 @@ The framework includes acknowledgment information in the data packet.
 | ACTUAL_CONNECTION_ID
 | For information only - when using a cached or failover client connection factory, contains the actual underlying connection id.
 |===
+
+
+[[ip-annotation]]
+=== Annotation-Based Configuration
+
+The following example from the samples respository is used to illustrate some of the configuration options when using
+annotations instead of XML.
+
+[source, java]
+----
+@EnableIntegration <1>
```

Comment:
The hint `<1>` looks cool!
Although all other code must be formatted without tabs.
Ins't it ?